### ISSUE TITLE
BidListResultsCard - Rename toggle method

### ISSUE THREAD

**Issue by burgwyn** [18F/State-TalentMAP#765]
Should probably just call `toggleBidPosition()` something else, like `removeBidPosition()` since it doesn't really toggle, it only removes

**Comment by mjoyce91** [comment]
Good point - the originating function can toggle, but from the user's bid list page it's only ever used to remove bids. Though, I could see us having some kind "undo" feature in the future, where you can re-add a bid if you accidentally remove it.
